### PR TITLE
Unifying the asocat lib handling. 

### PR DESCRIPTION
**Pull request by ljo**
Changing to correct name for jar file, now only in one place. Makes ant only builds happy.

### CODE REVIEW

**Comment by adamretter** [comment]
@ljo I think this is some tools that @dizzzz wrote to assist with the Ant build, right? As this wasn't in the classpath but the build was still working fine... do we actually still use these tools?

**Comment by ljo** [comment]
@adamretter Yes, we are using it for all extensions, the jarsigner, installer and for the macapp build.

**Comment by adamretter** [comment]
Erm... but I am doing a build with all extensions and the installed and the macapp without your PR and it seems to work?

**Comment by ljo** [comment]
@adamretter yes, that is probably becuase you are not using ant only.

**Comment by adamretter** [comment]
@ljo I am using eXist's `./build.sh` does that make a difference?

**Comment by ljo** [comment]
@adamretter yes, that works flawlessly for me too.

**Comment by adamretter** [comment]
@ljo so you mean invoking an external `Ant` binary?

**Comment by dizzzz** [comment]
afaik the ascot.jar is required to be able to build eXist-db ; note that there are some smart handles that prevent double download of jar files. It is e.g. used to download XAR files for the auto deploy directory

**Comment by ljo** [comment]
@adamretter Yes, when run with any ant binary it actually reads the given paths for the tasks and since the jar name is wrong the antclasspath gets empty.

File: build/scripts/build-impl.xml [hunk 1]
```diff
@@ -49,7 +49,7 @@
     <available file="key.store" property="key.store.present"/>
 
     <!-- additional set of ant tasks -->
-    <property name="asocat-exist.jar" location="${tools.ant}/lib/asocat-exist.jar"/>
+    <property name="asocat-exist.jar" location="${tools.ant}/lib/asocat-exist-1.0.jar"/>
```
Comment (dizzzz) [inline build/scripts/build-impl.xml]: so the jar needs to be renamed too? hmmm no it i already called 1.0?

**Comment by adamretter** [comment]
@ljo okay cool. Looks good then.

**Comment by ljo** [comment]
@dizzzz no, the jar file (tools/ant/lib/asocat-exist-1.0.jar) already has the name i changed to in the build-impl.xml since it doesn't match otherwise.

**Comment by ljo** [comment]
@adamretter ok, great.

**Pull request merged**